logic/minecraft/LegacyInstance.cpp

```diff
@@ -319,6 +320,16 @@ std::shared_ptr<ModList> LegacyInstance::texturePackList() const
 	return texture_pack_list;
 }
 
+std::shared_ptr<WorldList> LegacyInstance::worldList() const
```

Comment:
OneSixInstance::worldList and this worldList are basically identical, so the function should probably be moved up in the inheritance hierarchy.